test/EndToEndTests/Common/Microsoft.OData.E2E.TestCommon/Converters/BinaryPayloadConverter.cs

```diff
@@ -0,0 +1,34 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="BinaryPayloadConverter.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using Microsoft.OData.Edm;
+using System.Globalization;
+
+namespace Microsoft.OData.E2E.TestCommon.Converters
+{
+    public class BinaryPayloadConverter : ODataPayloadValueConverter

```

Comment:
Is this class used in other tests? If it's only used in the `PayloadValueConverterTests` then maybe it should be defined there?